Dockerfile

```diff
@@ -7,6 +7,8 @@ ARG BASE_IMAGE_TAG=jammy-latest
 ARG ECR_URI=${ECR_ACCOUNT_ID}.dkr.ecr-fips.${ECR_REGION}.amazonaws.com/${BASE_IMAGE_NAME}:${BASE_IMAGE_TAG}
 
 FROM ${ECR_URI} as docker-code-server-python
+# Set the R version to install, next is always the latest version: https://cdn.posit.co/r/versions.json
+ENV R_VERSION=next
```

Comment:
That's cool! I guess this means that every time we release a new version of the docker image, there'll be a new version of R installed. 

Hm, will that be a problem for pre-existing code clouds at all? I'm inclined to think no, since system packages aren't persisted in EFS, so there wouldn't be conflicts. And if you care about using a specific version, you should probably use a version manager of some sort.